integrationTests/vm/arwen/arwenvm/arwenVM_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !race
```

Comment:
Arwen should be ok now. Maybe remove these pre-processor lines along with the following TODO?
Valid on all files

Reply:
ok!

Reply:
seems not to be ready yet... undid the remove